docs/react-v9/contributing/dev-workflow.md

```diff
@@ -1,4 +1,94 @@
+
+### Building
+
+The inner development loop usually involves these steps:
+
+First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages.
+
+```
+cd fluentui
+yarn // this can take some time
+```
+
+Run `yarn start` and select your start up project.
+
+- @fluentui/public-docsite starts the public v8 documentation site.
+- @fluentui/public-docsite-v9 starts the internal v9 documentation storybook site.
+- Pick a specific component package if you intend to work on a v9 component directly.
+
+Most projects have hot module reloading, so file saves will trigger refreshes.
+
+### Iterating
+
+Make the changes you need, and commit along the way.
+
+```
+git status // Shows all changed files
+git add . // Stages the changed files
+git commit -m "Your brief message." // Makes the commits. You may notice a slight lag during committing as our linters work away.
+git push upstream // Pushes your changes to the central Fluent UI repo.
+```
+
+### Syncing with master
+
+It is strongly recommended that you rebase your branch onto (rather than merging with) with master.
+
+```
+git checkout master // Switches to master
+git pull upstream master // Syncs your local master with the latest version of master at the origin
+git checkout your-fancy-branch // Switches to your branch
+git rebase master // Tacks your commits onto the end of master.
+```
+
+Resolve any conflicts in your editor.
+
+`git push upstream` // Pushes your changes to the central Fluent UI repo.
+
+Creating _draft_ pull requests is often an easy way to keep track of your work without it being actively reviewed by others.
+
+In other cases, such as before checking in or running tests, you may need to run a full build (or build up to a certain package):
+
+- `yarn build` - build everything. It is good the run this weekly, or anytime you start a new project off master.
+- `yarn build --to package-name` - build up to a package.
+  - `yarn build --to @fluentui/react`
+
+You can also `cd` to any package under `packages/*` and run `yarn build` to build individual things, though keep in mind that this may require dependencies to be built first (using `build --to`).
+
+### Making a pull request
+
+Before creating a pull request, be sure to run `yarn change` and provide a high-level description of your change, which will be used in the release notes. When selecting a change type, use this guide:
+
+- Major - Don't do these without expressed agreement from the team. This is a very stable repo.
+- Minor - Adding new API surface area.
+- Patch - No change in API surface area.
+
+Make sure all your changes are committed, and run `yarn build`. This will compare your changes and detect if the publicly facing API has changed, and update the right docs accordingly. Commit this change.
```

Comment:
I would hate to tell people to run a global `yarn build` -- I know I never do unless something goes wrong 😅. How do you feel about suggesting a workspace-scoped one instead?
```suggestion
Make sure all your changes are committed, and run a build on the package that was updated, e.g. `yarn workspace @fluentui/react-components build` for changes to the v9 components. This will compare your changes and detect if the publicly facing API has changed, and update the right docs accordingly. Commit this change.
```

Reply:
Also, I think this should be moved before the `yarn change` section, since sometimes changes in APIs will need additional change files (that's caught me out multiple times 😂)